decidim-core/app/views/decidim/devise/registrations/new.html.erb

```diff
@@ -4,94 +4,72 @@
   <%= render "decidim/devise/shared/links" %>
 <% end %>
 
-<div class="wrapper">
-<div class="row collapse">
-  <div class="row collapse">
-    <div class="columns large-8 large-centered text-center page-title">
-      <h1><%= t(".sign_up") %></h1>
-      <p>
-        <%= t(".subtitle") %>
-      </p>
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
```

Comment:
The error messages should not show up on this page by default. They should only show up after there is actually an error recognized in the field as the current functionality. Or maybe this is some conflict with the Foundation JS or Foundation form helper?
![Form errors visible by default](https://user-images.githubusercontent.com/864340/183073603-217e2b97-a0ec-4ed4-98e9-fd86cd515029.png)

Also another thing here is that the error messages don't show up in the correct default color when I have cleared the organization's color settings. I can see that in the staging environment it shows up with the correct color.

Maybe we should add some default when the organization colors are not defined?

Reply:
> Or maybe this is some conflict with the Foundation JS or Foundation form helper?

Kind of. Foundation abide prevented to display any error on pristine forms. Mainly the "required" who triggers on load (quite annoying). We'll come back to this when we replace the abide behaviour (or figure something out)

> Also another thing here is that the error messages don't show up in the correct default color

Do not pay attention to this odds. It's about the problem I mentioned [here](https://github.com/decidim/decidim/pull/9455#discussion_r938755665). If you see something that doesn't fit, double check [in staging](https://github.com/decidim/decidim/pull/9455#discussion_r938755665)